tipocket-ctl/tpctl/debug.py

```diff
@@ -0,0 +1,135 @@
+import inspect
+
+import click
+
+HELP_STRING = """
+generate debug environment into .env file.
+
+    Run `source .env` to get commands.
+
+\b
+You can find argument {deploy-id} and {cluster-namespace} in slack notification:
+============ Slack Notification ======================
+argo workflow
+tpctl-hello-test-tpctl-q948q
+^^^ {DEPLOY ID} ^^^^^^^^^^^^
+...
+cmd
+/bin/hello ... -namespace="tpctl-hello-test-tpctl" ...
+               ^^^ {CLUSTER NAMESPACE} ^^^^^^^^^^^^^^^^^^
+=======================================================
+"""
+
+
+def generate_script(deploy_id, cluster_namespace):
+    variables = inspect.cleandoc("""
+    DEPLOY_ID={}
+    CLUSTER_NAMESPACE={}
+    """.format(deploy_id, cluster_namespace))
+
+    functions = inspect.cleandoc("""
```

Comment:
We'd better save the script content in a seperate `.sh` file.
It will be more convinient to debug and modify this script.

Reply:
Where should we place the script file?
Under the project root directory or in a separate directory like asset/env.sh?

I prefer a separate directory like asset/ or script/.

Reply:
> I prefer a separate directory like asset/ or script/

+1. I think we can have such layout

```
- tpctl/
  - prepare.py
  - ...
  - scripts/
     - xxx.sh
```

ref: https://stackoverflow.com/questions/6028000/how-to-read-a-static-file-from-inside-a-python-package